refactor python code

Wrench-Git opened the pull request:
refactor python code by ops

Wrench-Git committed:
refactor python code

refactor python code by ops

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/DeepLink-org/DeepLinkExt?pullRequest=133) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/DeepLink-org/DeepLinkExt?pullRequest=133) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/DeepLink-org/DeepLinkExt?pullRequest=133) it.</sub>

Wrench-Git committed:
update tests

Wrench-Git committed:
fix some bugs

Wrench-Git committed:
add support for InterEvo flash-attn

Wrench-Git committed:
refactor the import ways

Wrench-Git committed:
refactor the way of import deeplink_ext

Now apply_rotary is same with rotary_emb.apply_rotary and import deeplink_ext will do mock now.

Wrench-Git committed:
Merge branch 'refactor-python-code' of https://github.com/DeepLink-org/DeepLinkExt into refactor-python-code

Wrench-Git committed:
fix a bug which miss to patch rmsnorm
